### ISSUE TITLE
finetuning with correspondences

### ISSUE THREAD

**Issue by fedeloper** [TruongKhang/TopicFM#12]
Can the model be fine-tuned with images and their corresponding labels, without using a depth map?

**Comment by TruongKhang** [comment]
What do you mean by "corresponding labels" here? semantic segmentation labels?
